src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
```

Comment:
Apparently it's fault of OS X apps building with ancient OS X-shipped OpenSSL versions causing trouble.